xml/dns_versionbind.xml

```diff
@@ -0,0 +1,685 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  This fingerprint file matches the text string response from a DNS
+  version.bind request.
+
+  For example, the string 'dnsmasq-2.76-1-ubnt2' emitted by the command below:
+
+  $ nslookup -type=txt -class=chaos VERSION.BIND <dns_server> | grep VERSION.BIND | cut -d\" -f2
+  dnsmasq-2.76-1-ubnt2
+
+-->
+<fingerprints matches="dns_versionbind" protocol="dns" database_type="service" preference="0.750">
```

Comment:
Perhaps, this has been an annoyance for me in the past.  The files are typically named one way and the `matches` another (underscore vs dots).  Since only some of the files have a `matches` section its hit or miss as which name works when you are calling Recog's databases programatically.  My preference would be to pick one naming scheme and stick with it or, failing that, implement a `matches` section for all files using the dot syntax. Unfortunately we'd potentially implement a breaking change for someone's workflow if we did that.  In this case I've made it simple by setting both values to be the same.